tools/packaging/kata-deploy/local-build/dockerbuild/install_oras.sh

```diff
@@ -37,9 +37,9 @@ fi
 oras_tarball="oras_${oras_required_version#v}_linux_${arch}.tar.gz"
 
 echo "Downloading ORAS ${oras_required_version}"
-sudo curl -OL https://github.com/oras-project/oras/releases/download/${oras_required_version}/${oras_tarball}
+curl -OL https://github.com/oras-project/oras/releases/download/${oras_required_version}/${oras_tarball}
 
 echo "Installing ORAS to ${install_dest}"
-sudo mkdir -p "${install_dest}"
-sudo tar -C "${install_dest}" -xzf "${oras_tarball}"
-sudo rm -f "${oras_tarball}"
+mkdir -p "${install_dest}"
```

Comment:
You need `sudo` for creating a dir under `/usr/`

Reply:
AFAUI, this is run during `docker build` as root.

Reply:
> AFAUI, this is run during `docker build` as root.

Then we're good.

Reply:
@mythi , I use install_oras.sh when I set-up my test machines manually, so I would prefer to keep the sudo here. 

But it's OK with me if you and other reviewers think we should remove it.

Reply:
for me the problematic part was `sudo curl ...`. would it work if I just drop `sudo` from that part?

Reply:
@mythi , yes, I believe dropping sudo for curl would work for users like me too.

Reply:
updated accordingly.